### PR TITLE
feat: added support contact dropdown type

### LINKED ISSUE #591: Add support contact section and drop down
**User Story:**
As a round operator
I want to configure support contact details
So that applicants know how to reach me with questions

**Acceptance Criteria**
GIVEN that I am on step one of Round Creation
WHEN I look at the page
THEN I see the "Where can applicants...?" section below the Round Name section and above the Dates section
AND see the empty state of the Support Input field

GIVEN that I am on step one of Round Creation
WHEN I click on the Support Input drop down
THEN I see the four contact types

GIVEN that I have the Support Input drop down open
WHEN I select a contact type
THEN the drop down closes
AND that contact type is shown in the field

GIVEN that I have not selected one of the four contact types in Support Input
WHEN I click next
THEN I see an error message that I must select a contact type

**Product & Design Links:**
[Figma](https://www.figma.com/file/K31Tf7tSn10BuN65VC6YLc/Round-Manager---Hi-Fidelity-Designs?node-id=2359%3A15064)

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by hmrtn**
fixes #591

### CODE REVIEW

**Review by vacekj — COMMENTED** [review]
Re: the todos - did you mean to leave them in for now and take care of them in a later ticket?

File: packages/round-manager/src/features/round/RoundDetailForm.tsx [hunk 1]
```diff
@@ -895,6 +909,206 @@ function MatchingCap(props: {
...
+                          )
+                      )}
+                    </Listbox.Options>
+                  </Transition>
+                </div>
+                {props.errors.roundMetadata?.support?.type && (
+                  <p className="mt-2 text-xs text-pink-500">
+                    {
+                      "You must select a support type."
+                      // TODO: Use YUP for error message
```
Comment (thelostone-mc) [inline packages/round-manager/src/features/round/RoundDetailForm.tsx:1040]: Should we update this right now to keep it consistent ?

**Review by melissa-neira — COMMENTED** [review]
Yeah, seeing the same issue as @nfrgosselin

File: packages/round-manager/src/features/round/RoundDetailForm.tsx (hunk 1 above)
Comment (hmrtn) [reply packages/round-manager/src/features/round/RoundDetailForm.tsx:1040]: Yes - however I could not for the life of me figure out why it was not working as intended. Perhaps the nested yup objects require a different approach?

**Comment by hmrtn** [comment]
@nfrgosselin @thelostone-mc This may deserve its own issue - it is affecting the staging and release deployments.

File: packages/round-manager/src/features/round/RoundDetailForm.tsx (hunk 1 above)
Comment (thelostone-mc) [reply packages/round-manager/src/features/round/RoundDetailForm.tsx:1040]: @chibie got any ideas here ?  you've got a solid understanding of how yup works

**Comment by hmrtn** [comment]
whoops didn't mean to close this

**Review by nfrgosselin — APPROVED** [review]
Approving as-is. Dropping a separate bug for the resizing issue #610

**Pull request merged**